resources/tag_models/page.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
I'm not fixed on this either, especially as they are very similar anyway.

The current thinking is we have a "page" at each end of the pipeline, why try to fit it into one, to expand it out again later.
The other idea is that a page has more values, whereas a event typically has one value.
Finally, a page is special when it comes to reporting, it is a different "type" of thing compared to an event.